src/main/kotlin/org/rust/lang/core/psi/impl/RustRawStringLiteralImpl.kt

```diff
@@ -1,12 +1,47 @@
 package org.rust.lang.core.psi.impl
 
 import com.intellij.psi.tree.IElementType
-import org.rust.lang.core.psi.RustLiteral
 import org.rust.lang.core.psi.RustLiteralTokenType
 
-class RustRawStringLiteralImpl(type: IElementType, text: CharSequence) : RustLiteral.Text(type, text) {
+class RustRawStringLiteralImpl(type: IElementType, text: CharSequence) : RustTextLiteralImplBase(type, text) {
+    override val value: String?
+        get() = valueString
+
     override fun toString(): String = "RustRawStringLiteralImpl($tokenType)"
 
+    private var hashes = 0
+
+    override fun locateOpenDelim(start: Int): Int {
+        assert(text[start] == '#' || text[start] == '"') { "expected open delim" }
```

Comment:
I think it's better to use `check` instead of assert. asserts are disabled by default, and check is always on.